examples/metrics_simple/README.md

```diff
@@ -1,86 +1,108 @@
 # Simple Metrics Example
 
-This example initializes the metrics pipeline with 2 different instrument types.
-Here are more detailed explanations of each part.
+This example initializes the metrics pipeline with 3 different instrument types -
```

Comment:
'-' character at the end, did you mean ":" instead ?

Reply:
Yeah should have been ":", changed it.